.github/workflows/build.yml

```
@@ -342,8 +341,7 @@ jobs:
           version: ${{needs.configuration.outputs.version}}
           sa_key: ${{ secrets.GCP_CREDENTIALS }}
           previewctl_hash: ${{ needs.build-previewctl.outputs.previewctl_hash }}
-          wsmanager_mk2: ${{needs.configuration.outputs.with_ws_manager_mk2}}
-          with_dedicated_emu: ${{needs.configuration.outputs.with_dedicated_emulation}}
+          with_dedicated_emulation: ${{needs.configuration.outputs.with_dedicated_emulation}}
```

```suggestion
          with_dedicated_emu: ${{needs.configuration.outputs.with_dedicated_emulation}}
```
the action still expects the `with_deciated_emu` input (`deploy-gitpod/entrypoint.sh` uses the `INPUT_WITH_DEDICATED_EMU` env var)